certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6000 @@
+                                }
+                              }
+                            },
+                            "description": "Layout group formats."
+                          }
+                        }
+                      },
+                      "description": "Layout groups."
+                    },
+                    "parameters": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "example": {
+                            "type": "string",
+                            "description": "Example.",
+                            "title": "Example"
+                          },
+                          "technicalIdentifier": {
+                            "type": "string",
+                            "description": "Technical identifier.",
+                            "title": "Technical id"
+                          },
+                          "schema": {
+                            "type": "string",
+                            "description": "Schema.",
+                            "title": "Schema"
+                          },
+                          "sensitive": {
+                            "type": "boolean",
+                            "description": "Sensitive.",
+                            "title": "Sensitive",
+                            "enum": [
+                              true,
+                              false
+                            ]
+                          },
+                          "required": {
+                            "type": "boolean",
+                            "description": "Required.",
+                            "title": "Required",
+                            "enum": [
+                              true,
+                              false
+                            ]
+                          },
+                          "value": {
+                            "type": "string",
+                            "description": "Value.",
+                            "title": "Value"
+                          },
+                          "placeholderEnabled": {
+                            "type": "object",
+                            "properties": {
+                              "commandSet": {
+                                "type": "boolean",
+                                "description": "Command Set.",
+                                "title": "Command Set",
+                                "enum": [
+                                  true,
+                                  false
+                                ]
+                              },
+                              "commandBlock": {
+                                "type": "boolean",
+                                "description": "Commmand Block.",
+                                "title": "Command Block",
```

Comment:
Can you please correct the spelling "Commmand"

Reply:
Fixed